test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get home_index_url
+    assert_response :success
+  end
+
+end
```

Comment:
I suggest to not use autogenerated tests.

This test only checks 200OK and checks nothing about page content, we better will add RSpec and write own tests